docs/changelog.md

```diff
@@ -1,3 +1,28 @@
+## v1.6.0next [2019-08-XX]
+
+#### Added
+
+* Allow login using certificate
+* Interface to install plugins on South, North, Filters and Notification page
+* Display the plugin name and version on South and North page
+* Support for password configuration items
+* JSON editor for json data
+* Supprot to update python script 
+* New log tabs for Notification and Packages under logs section
+* Link added in alert to show failure logs 
+
+#### Changed 
+
+* Update asset reading page to show textual data
+
+#### Fixed
+
+* Fix upload script issue on notification
+* Fixed wrong time on graph data for the past reading
+* Fixed save multiple filter configuration at once
+* Fixed csv formate to download for asset readings
```

Comment:
Fixed csv formate for asset readings download

Reply:
csv format